Define which internal dependencies should be built.
Building them with EB and linking them didn't work.
Author: Caspar van Leeuwen (SURFsara)
Adapted from: FSL-6.0.1_Build_extras.patch
Original author: Davide Vanzo (Vanderbilt University)
--- fsl.orig/extras/build	2019-11-15 10:48:52.249162071 +0100
+++ fsl/extras/build	2019-11-15 17:19:51.859063451 +0100
@@ -106,6 +106,8 @@
 fi
 PROJECTS="${PROJECTS} libgd libgdc libprob libcprob newmat cprob newran fftw"
 PROJECTS="${PROJECTS} boost libxml2-2.9.2 libxmlpp libsqlite libnlopt ../include/armawrap/dummy_newmat"
+# For EasyBuild
+PROJECTS="libgdc libprob libcprob cprob newran ../include/armawrap/dummy_newmat"
 for projname in $PROJECTS; do
     if [ -d $FSLESRCDIR/$projname ] ; then
        buildIt $FSLESRCDIR $projname 1
